### PR TITLE
bugfix

### PR DESCRIPTION
**Pull request by SNESTracker**
simple fix here. pretty self-explanatory.

### CODE REVIEW

File: a.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 oeeuuuuu code!!!
 
 
-A BUG IN THE CODE
+NOw THE BUG IS FIXED
```
Comment (bazz1tv) [inline a.txt]: looks good, but the 'w' needs to be capitalized.

**Comment by bazz1tv** [comment]
LGTM. accepted. Here's your :cake:

**Comment by bazzinotti** [comment]
test

**Comment by bazzinotti** [comment]
test

**Comment by bazzinotti** [comment]
test

File: a.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 oeeuuuuu code!!!
 
 
-NOw THE BUG IS FIXED
+NOW THE BUG IS FIXED
```
Comment (bazzinotti) [inline a.txt]: sample comment

File: a.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 oeeuuuuu code!!!
 
 
-A BUG IN THE CODE
+NOW THE BUG IS FIXED
```
Comment (bazzinotti) [inline a.txt]: test

**Pull request merged**